central/notifiers/cscc/cscc_test.go

```diff
@@ -0,0 +1,87 @@
+package cscc
+
+import (
+	"context"
+	"testing"
+
+	"github.com/gogo/protobuf/types"
+	"github.com/golang/mock/gomock"
+	clusterMocks "github.com/stackrox/rox/central/cluster/datastore/mocks"
+	"github.com/stackrox/rox/central/notifiers/cscc/findings"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestWithFakeCSCC(t *testing.T) {
+
+	const (
+		projectKey = "FJ"
+	)
+
+	sourceID := "organizations/847401270788/sources/394194234514334994"
+
+	s := &storage.Notifier{
+		Name:         "FakeSCC",
+		UiEndpoint:   "https://central.stackrox",
+		Type:         "scc",
+		LabelDefault: projectKey,
+		Config: &storage.Notifier_Cscc{
+			Cscc: &storage.CSCC{
+				ServiceAccount: "test_service_account",
+				SourceId:       sourceID,
+			},
+		},
+	}
+
+	cluster := &storage.Cluster{
+		Id:   "test_id",
+		Name: "test_name",
+		Status: &storage.ClusterStatus{
+			ProviderMetadata: &storage.ProviderMetadata{
+				Zone: "test_zone",
+				Provider: &storage.ProviderMetadata_Google{
+					Google: &storage.GoogleProviderMetadata{
+						Project:     "test_project",
+						ClusterName: "test_cluster",
```

Comment:
nit : Although probably it doesn't matter for this test, you can keep the ClusterName here same as the cluster.Name set above, just for consistency.

Reply:
done